classroom.rb

```diff
@@ -0,0 +1,46 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(name, age, parent_permission: true, classroom: nil)
+    super(name, age, parent_permission)
+    @classroom = classroom
+    classroom&.add_student(self)
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] Please kindly ensure that there is a many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. You can achieve this by creating `add_rentals` method.
- [ ] Please kindly also create same in the `Person` class.